api/streaming/streaming-session.ts

```diff
@@ -140,17 +142,15 @@ export class StreamingSession extends Subscribed {
       )
     }
 
-    this.onCallEnded.subscribe(() => ff.stop())
-
-    ff.writeStdin(inputSdp)
+    this.ff.writeStdin(inputSdp)
 
     // Activate the stream now that ffmpeg is ready to receive
     await this.activate()
   }
 
   private callEnded() {
     this.unsubscribe()
-    this.onCallEnded.next()
+    this.ff?.stop()
```

Comment:
I think a simpler solution might be to add a `private hasEnded = false` and return early here if it has, preventing the internals of `callEnded` from executing twice.  Any thoughts on that approach?